content/02.intro.md

```diff
@@ -1 +1,10 @@
 ## Introduction
+
+Deep learning (DL), a subfield of machine learning (ML), is exploding in popularity and is increasingly being used for biological data analysis [@doi:10.1098/rsif.2017.0387].
```

Comment:
Maybe extending

"a subfield of machine learning (ML)" => 
"a subfield of machine learning (ML) that focuses on deep artificial neural networks"

since the audience will likely be mostly biologists not computer scientists.